src/components/Search/Filters/Control/DesktopControl.tsx

```diff
@@ -11,6 +11,7 @@ import AccessibilityInput from "../AccessibilityInput";
 import DistanceInput from "../DistanceInput";
 import ControlToggles from "./DesktopControlToggles";
 import AgeGroupInput from "../AgeGroupInput";
+import { useLocation, useNavigate } from "react-router-dom";
```

Comment:
seems like useEffect, useLocation, and useNavigate aren't used?

Reply:
yup this was from a much more involved and very unnecessary first attempt at a fix